pkg/cluster_agent/forwarder.py

```diff
@@ -12,20 +12,37 @@ class Forwarder:
     """
     A generic KubernetesEvent forwarder
     """
-
-    MAX_EVENTS_TO_READ = 300
+    DEFAULT_MAX_WORKERS = 1
+    DEFAULT_MAX_EVENTS_TO_READ = 100
 
     def __init__(
             self,
             events_manager: EventsManager,
-            events_sender: EventsSender
+            events_sender: EventsSender,
+            max_workers: int = DEFAULT_MAX_WORKERS,
+            max_events_to_read: int = DEFAULT_MAX_EVENTS_TO_READ
     ):
         """
         :param events_manager: used to read from events
         :param events_sender: used to send read events to
+        :param max_workers: to forward read events
+        :param max_events_to_read: to read from the events_manager
         """
         self.events_manager = events_manager
         self.events_sender = events_sender
+        if max_workers < 1:
+            raise ValueError("Invalid workers count value, must be > 0")
+        self.max_workers_count: int = max_workers
```

Comment:
Do we need the int definition here when the function param already mentions that max_workers is int?
if yes, then we should also add it to max_events_to_read for example

Reply:
not mandatory, but will add for max_events